res/controllers/M-Audio-Xponent-Advanced-scripts.js

```diff
@@ -18,23 +18,23 @@ MaudioXponent.config = {
     pflMode : 0,            // 0 = Independent, 1 = Toggle
     syncFlashMode : 0,      // 0 = Off, 1 = Pulse, 2 = Toggle
     vuMeterMode : 1,        // 0 = Master, 1 = Channel
-    tapDuration : 250,      // Maximum touch duration for wheel tap (in milliseconds)
```

Comment:
This does not matter much, but IMO it is good practice to leave a trailing comma after the last key-value pair in an object literal. If you ever add more to that object literal, it's easy to forget to add that comma.